### ISSUE TITLE
Console.log des Buttons wird nicht angezeigt

### ISSUE THREAD

**Issue by MaximilianoRottenanger** [JirkaDellOro/EIA2-Inverted#460]
Hallo Jirka,
leider wird bei mir der console.log("Button Geklickt") nicht in der konsole angezeigt. Ich habe schon geschaut ob der Button richtig ausgewählt wurde aber weiss nicht warum es nicht geht.

Hier der Link zur ts datei: https://github.com/MaximilianoRottenanger/EIA2_Repository/blob/main/A02_EventInspector/inspector.ts

Mit freundlichen Grüßen,
Maximiliano

**Comment by KohlerAl** [comment]
Hallo Maximiliano, hast du einen Link zu github pages, damit wir den Code testen können?

**Comment by MaximilianoRottenanger** [comment]
Hallo, leider weiss ich nicht wie das geht. eigentlich sollte ich genau so einen link haben, der eine html page öffnet und nicht den github ordner. ich habe github damals mit alexander eingerichtet aber haben das wohl vergessen. SO sieht meine einstellung bei pages aus: 
![image](https://github.com/user-attachments/assets/26d9625b-efe4-4b7b-8453-9e0a92266b6f)

**Comment by KohlerAl** [comment]
Genau, das ist richtig. Jetzt nimmst du den Link von der Einstellungsseite und dann öffnest du die HTML Datei von der Abgabe auf Github. Da müssten oben rechts drei Punkte sein, mit der Option "Copy Path". Kopier das und häng es hinten an den Link von den Pages-Einstellungen an. Dann hast du den Link, mit dem wir die Abgabe hier anschauen können (und auch den Link, den du für die Abgabe auf Felix brauchst)

**Comment by MaximilianoRottenanger** [comment]
https://maximilianorottenanger.github.io/EIA2_Repository/A02_EventInspector/inspector.html

jetzt hat es geklappt, danke!. jetzt verstehe ich auch was jirka bei meiner letzten abgabe gemeint hat... ich werde den link von der letzten abgabe dann nochmal auf felix hochladen. ich hoffe das zählt jetzt nicht als nicht abgegeben... er hat warscheinlich nur die diagramme gesehen.

**Comment by KohlerAl** [comment]
Perfekt, das ist der richtige Link! Wenn du irgendwas aus dem Feedback nicht verstehst, kannst du uns auch immer Fragen :)

**Comment by KohlerAl** [comment]
Wegen deinem Problem: Das Kästchen, das deinem Mauszeiger folgt, hat kein Offset, das bedeutet du klickst immer auf das Kästchen und nie auf den Button. Wenn du in deinem Script in den Zeilen 24 und 25 jeweils noch ein kleines bisschen offset auf die Position addierst, dann sollte es funktionieren :)

**Comment by MaximilianoRottenanger** [comment]
Hat geklappt. super, danke !
